src/Microsoft.Azure.WebJobs.Host/TypeUtility.cs

```diff
@@ -98,6 +98,25 @@ internal static Type UnwrapTaskType(Type type)
             return null;
         }
 
+        internal static TAttribute GetResolvedAttribute<TAttribute>(ParameterInfo parameter) where TAttribute : Attribute
+        {
+            var attribute = parameter.GetCustomAttribute<TAttribute>();
+
+            var storageAttribute = attribute as StorageAttribute;
```

Comment:
1. You could still accomplish that goals using the existing StorageAccountAttribute  (and not introducing a new StorageAttribute) 
2. What about ServiceBus? Do we copy this pattern for SB too? 
3. What about tooling? Does it special case? Actually, it'd be great if tooling can just call this method, and thne it matters much less. 

---
In reply to: [107949364](https://github.com/Azure/azure-webjobs-sdk/pull/1084#discussion_r107949364) [](ancestors = 107949364)